crhchttp/http_cloud_dot_redhat.go

```diff
@@ -105,7 +104,7 @@ func SetupRequest(authConfig *AuthConfig, contentType, method, uri string, body
 	default:
 		log.Info("Request using token authentication")
 		req.Header.Set("Authorization", fmt.Sprintf("Bearer %s", authConfig.BearerTokenString))
-		req.Header.Set("User-Agent", fmt.Sprintf("cost-mgmt-operator/%s cluster/%s", authConfig.OperatorCommit, authConfig.ClusterID))
+		req.Header.Set("User-Agent", fmt.Sprintf("koku-mertics-operator/%s cluster/%s", authConfig.OperatorCommit, authConfig.ClusterID))
```

Comment:
This cannot be changed unless we submit a PR to the uhc-auth-proxy repo to change our prefix name https://github.com/RedHatInsights/uhc-auth-proxy/pull/5

Reply:
reverted back to `cost-mgmt-operator`